### PR TITLE
Add rescueeer

### PR DESCRIPTION
**Pull request by oldremez**
rescueeer

neutron PR (genesis): https://github.com/neutron-org/neutron/pull/233
tests: https://github.com/neutron-org/neutron-integration-tests/pull/151

### CODE REVIEW

File: contracts/misc/rescueeer/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Neutron Credits Vault
```
Comment (oopcode) [inline contracts/misc/rescueeer/README.md:1]: huh?

File: contracts/misc/rescueeer/src/contract.rs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    Ok(Response::default().add_messages(msgs))
+}
+
+pub fn execute_transfer_admin(
+    deps: DepsMut,
+    env: Env,
+    _info: MessageInfo,
+    addresses: String,
```
Comment (sotnikov-s) [inline contracts/misc/rescueeer/src/contract.rs:58]: address**es** -> address

File: contracts/misc/rescueeer/src/error.rs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+#[derive(Error, Debug)]
+pub enum ContractError {
+    #[error("{0}")]
+    Std(#[from] StdError),
+
+    #[error("Unauthorized")]
+    Unauthorized {},
+
+    #[error("End of life haven't reached yet")]
```
Comment (sotnikov-s) [inline contracts/misc/rescueeer/src/error.rs:12]: ha**ven't** -> ha**sn't**

File: contracts/misc/rescueeer/src/contract.rs (hunk 2 above)
Comment (oldremez) [reply contracts/misc/rescueeer/src/contract.rs]: fixed

**Review by oopcode — APPROVED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**